clang/test/SemaSYCL/msvc-fixes.cpp

```diff
@@ -0,0 +1,7 @@
+// RUN: %clang_cc1 -triple spir64-unknown-windows-sycldevice -fsycl-is-device -aux-triple x86_64-pc-windows-msvc  -fsyntax-only -verify %s
```

Comment:
@bader I corrected the test case here as you requested

Reply:
@mibintc, thanks a lot!
This looks like something SYCL driver should use.

Reply:
@bader you mean you want the sycl driver to add the aux triple on windows compilations?

Reply:
@mibintc, yes. The same way as the driver works on Linux.
@mdtoguchi, doesn't driver pass -aux-triple on Windows?

Reply:
> @mibintc, yes. The same way as the driver works on Linux.
> @mdtoguchi, doesn't driver pass -aux-triple on Windows?

Yes, -aux-triple is added for Windows.